### DISCUSSION TITLE
Add an explicit method (link/button/selectbox) for switching grid views

### DISCUSSION THREAD

**Discussion by jacklinke** [djangopackages/djangopackages#794] [Feature Ideas]
**Is your feature request related to a problem? Please describe.**
Grids can sometimes be challenging to mentally process, particularly when a grid contains many packages *and* many features. On a typical landscape oriented monitor, you end up scrolling left and right and up and down to fully grok which packages have which features.

For instance, here is what you see if you go to the "Trees and Graphs" package grid. It's what I've been used to over the years. Good simple landscape view, but you have to scroll right to see additional packages.  As mentioned, on a typical (non-vertical) monitor or screen, you have to scroll up and down to see which features each package has. And since the feature labels at the left aren't sticky, you end up scrolling up and down, left and right to get a good understanding of which package has which features.

https://djangopackages.org/grids/g/trees-and-graphs/
	
Today I somehow stumbled upon another grid view option that I wasn't aware even existed, and it's glorious. I can simply scroll down and see everything I want to see. Still not perfect (if we could get those headers to be sticky, that'd be icing on the cake), but for large grids this view can be *so much* clearer and easy to use!

https://djangopackages.org/grids/g/trees-and-graphs/landscape/?simpleGrid=true

Unfortunately, there seems to be no button or selection to see alternative grid views from the standard grid view. And while I've been using djangopackages for a few years now, I've never seen any notice on the site that alternative views even _existed_, much less how to switch them up.

**Describe the solution you'd like**	

- Continue to default to the standard landscape view, but add a method of selecting the desired view
- Add some documentation somewhere (maybe just a simple FAQ entry?) describing the various grid views

**Comment by jefftriplett** [comment]
I'm hoping to look more into this, this weekend. 

Short term, I'd like to prune out projects that are outdated, abandoned, deprecated, archived, etc. Once we have a better handle on this, it might even be worth rethinking the display completely and only showing the top-3, 5, or 10 grids. I personally don't want GitHub Stars to be the main driver though. 

Longer-term, I think we have an issue with some Grids are basically groups of "Framework" Categories that have grown too big. Check out https://djangopackages.org/open/ and you'll see that DRF, Django-CMS, Wagtail CMS, and Mezzanine have outgrown the concept of a grid. 

Whereas the Authentication Grid has a lot of libraries that might be better served in a more specific sub-category of Auth like 2FA. 
 https://djangopackages.org/grids/g/authentication/

Another short-term solution might be to default people to `simpleGrid` if the Grid has more than 20 packages.
